go/tools/builders/stdliblist.go

```diff
@@ -279,12 +280,19 @@ func stdliblist(args []string) error {
 
 	encoder := json.NewEncoder(jsonFile)
 	decoder := json.NewDecoder(jsonData)
+	pathReplaceFn := func (s string) string {
+		if strings.HasPrefix(s, absCachePath) {
+			return strings.Replace(s, absCachePath, filepath.Join("__BAZEL_EXECROOT__", *cachePath), 1)
```

Comment:
@fmeum instead of doing this, do you think it's safe to simply find the position of "execroot" replace the whole prefix with `__BAZEL_EXECROOT__`?

Reply:
will decide later